runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/AbstractDokkaTask.kt

```diff
@@ -18,7 +18,7 @@ abstract class AbstractDokkaTask : DefaultTask(), Configurable {
     var outputDirectory: String = defaultDokkaOutputDirectory().absolutePath
 
     @Input
-    override val pluginsConfiguration: Map<String, String> = mutableMapOf()
+    override val pluginsConfiguration: MutableMap<String, String> = mutableMapOf()
```

Comment:
(Comment to this property, not this PR)
 I'm not sure if Gradle DSL allows `val` s as it has to create a decorator for this property

Reply:
Added configuration for this to the BasicGroovyIntegrationTest and it seems to accept it 👍